netclient/global_settings/globalsettings.go

```diff
@@ -0,0 +1,5 @@
+package global_settings
+
+// globalsettings - settings that are global in nature.  Avoids circular dependencies between config loading and usage.
+
+var PublicIPServices map[string]string
```

Comment:
missing comment defining what PublicIPServices is, should be // PublicIPServices - <description>
likewise for any public func or variable

Also, I don't believe this variable is ever initialized -> make(map[string]string)